build-logic/publishing/src/main/kotlin/testng.maven-publish.gradle.kts

```diff
@@ -0,0 +1,67 @@
+plugins {
+    `maven-publish`
+    id("testng.signing")
+}
+
+object This {
+    const val version = "7.5.0-SNAPSHOT"
```

Comment:
Is it the common way to declare const in gradle?
I'm not feeling it good but I've never found a better way to do it.

Reply:
The version is to be declared in `gradle.properties`
The rest should be moved elsewhere as well, and `object This` is an anti-pattern for Gradle scripts.

Reply:
Feel free to change it too.